ckanext/canada/templates/internal/page.html

```diff
@@ -32,7 +32,35 @@
     </div></div>
   {% endblock content %}
 
-{%- block adobe_analytics_footer -%}{%- endblock -%}
-{%- block google_analytics_footer -%}
-  <script type="text/javascript" src="{{ h.url_for_static('/registry_ga_event_tracking.js') }}" ></script>
+{%- block scripts -%}
+  {{ super() }}
+  {%- block adobe_analytics_footer -%}{%- endblock -%}
+  {%- block google_analytics_footer -%}
+    <script type="text/javascript" src="{{ h.url_for_static('/registry_ga_event_tracking.js') }}" ></script>
+  {%- endblock -%}
+
+  {%- block session_timeout_footer -%}
+  {% if c.userobj %}
+    <section id="timeout" class="mfp-hide modal-dialog modal-content overlay-def">
+      <header class="modal-header">
+        <h2 class="modal-title">{{ _('Your user sessions has timed out due to inactivity') }}</h2>
+      </header>
+      <div class="modal-body">
+        {{ _('If you are currently entering data on this page, you may lose it when submitted. To avoid this, you can click on the log in button below to log in again in a new tab without losing your progress and you can then keep working on this page. If you have any issues please contact <a href="mailto:open-ouvert@tbs-sct.gc.ca">open-ouvert@tbs-sct.gc.ca</a>.') | safe }}
+      </div>
+      <div class="modal-footer">
+        <a href="{% url_for controller='user', action='login' %}" target="_blank" class="btn btn-default btn-sm">
+          <span class="fa fa-sign-in"></span>
+          <span class="text">{{ _('Log in') }}</span>
+        </a>
+      </div>
+    </section>
+    <script type="text/javascript" src="{{ h.url_for_static('/session_timeout.js') }}" ></script>
+    <script>
+      $(document).ready(function() {
+        timeoutPop({{ h.get_timeout_length() }});
+      });
+    </script>
```

Comment:
this section needs to be disabled when there is no timeout configured.

Reply:
Oh true. I disabled the whole block instead (including the html portion).